profiles/daily_trading/profile.json

```diff
@@ -0,0 +1,43 @@
+{
+    "config": {
+        "crypto-currencies": {
+            "Bitcoin": {
+                "enabled": true,
+                "pairs": [
+                    "BTC/USDT"
+                ]
+            }
+        },
+        "exchanges": {
+            "binance": {
+                "enabled": true
+            }
+        },
+        "trader": {
+            "enabled": false,
+            "load-trade-history": false
+        },
+        "trader-simulator": {
+            "enabled": true,
+            "fees": {
+                "maker": 0.1,
+                "taker": 0.1
+            },
+            "starting-portfolio": {
+                "BTC": 10,
+                "USDT": 1000
+            }
+        },
+        "trading": {
+            "reference-market": "USDT",
+            "risk": 0.5
+        }
+    },
+    "profile": {
+        "avatar": "default_profile.png",
+        "description": "DailyTrading is a low risk versatile profile that catches buy and sell opportunities using RSI and moving average indicators.",
```

Comment:
```suggestion
        "description": "Profile using the Daily Trading Mode which is is a low risk versatile profile that catches buy and sell opportunities using RSI and moving average indicators.",
```

Reply:
Why not "DailyTrading", it's the name of the profile

Reply:
right 👍